libs/wire-api/src/Wire/API/Routes/Version.hs

```diff
@@ -179,7 +183,88 @@ type VersionAPI =
         :> Get '[JSON] VersionInfo
     )
 
-versionSwagger :: S.Swagger
-versionSwagger = toSwagger (Proxy @VersionAPI)
+data VersionAPITag
+
+-- Development versions
 
 $(genSingletons [''Version])
+
+$( promote
+     [d|
+       isDevelopmentVersion :: Version -> Bool
+       isDevelopmentVersion V0 = False
+       isDevelopmentVersion V1 = False
+       isDevelopmentVersion V2 = False
+       isDevelopmentVersion V3 = False
+       isDevelopmentVersion _ = True
```

Comment:
Shouldn't `V4` become `False` in this PR?

Reply:
I don't think so because the plan is to first have V4 and V5 co-exist as development versions, and only in the next step/PR we finalize V4.

Reply:
That's right, v4 is still development. That's kind of the point of this PR. But that reminds me that we don't need to promote this function any more. It was just part of a failed attempt at removing some of the code duplication.